src/foam/nanos/auth/Group.js

```diff
@@ -185,7 +185,7 @@ if ( session != null ) {
       //populate AppConfig url with request's RootUrl
       HttpServletRequest req = x.get(HttpServletRequest.class);
 
-      if ( config.getEnableForceHttps() ) {
+      if ( config.getForceHttps() ) {
```

Comment:
And regarding lines 172 - 208 of Group.js:
Why is the whitespace formatting inconsistent with the rest of the file?  Usually newly inserted code should match the style set by the existing file (unless what was there before was really bad and inconsistent itself).

Reply:
re: styling, we have a style guide and we recommend to update a file touched to the new styling, but hopefully in a different PR.